api/query/params.go

```diff
@@ -39,14 +37,14 @@ func (p *Params) Decode(u *url.URL) error {
 		switch key {
 		case "tag":
 			for _, value := range values {
-				tag := strings.Split(value, ":")
-				if len(tag) != 2 {
+				if i := strings.Index(value, ":"); i != -1 {
```

Comment:
@jmassara can we make this simpler w/
 `if strings.Index(value, ":") != -1 {`

Reply:
I need the index value in order to slice the tag.